apps/python/scdbstrip.py

```diff
@@ -572,7 +572,9 @@ def validateParameters(self):
                 and self._hoursToKeep is None
                 and self._minutesToKeep is None
             ):
-                self._daysToKeep = 30
+                error.write("ERROR: at least one of datetime, days, hours,"
```

Comment:
Or `daterange` soon ;)

Reply:
if you merge this, I will rebase the PR of ` datarange` option and update the message